### PR TITLE
docs: specific list backup + restore

### PR DESCRIPTION
**Pull request by d1nuc0m**
Specify how to create a backup when a non-default finding list is used (see #140).

Add instructions to restore a backup.

### CODE REVIEW

**Comment by d1nuc0m** [comment]
@0x6d69636b please verify if this is correct or if I misunderstood some switches

File: README.md [hunk 1]
```diff
@@ -148,6 +148,22 @@ Invoke-HardeningKitty -Mode Config -Backup
 
 Please test this function to see if it really works properly on the target system before making any serious changes. A Schrödinger's backup is dangerous.
 
+##### Non-default finding list
```
Comment (0x6d69636b) [inline README.md:151]:
```suggestion
##### Non-Default Finding List

```

File: README.md [hunk 2]
```diff
@@ -148,6 +148,22 @@ Invoke-HardeningKitty -Mode Config -Backup
...
 
+##### Non-default finding list
+
+Note that if _-FileFindingList_ is not specified, the backup is referred to the default finding list. Before deploying a _specific_ list in _HailMary_ mode, always create a backup _referred to that specific list_.
+
+```powershell
+Invoke-HardeningKitty -Mode Config -Backup -BackupFile ".\myBackup.csv" -FileFindingList ".\list\{list}.csv"
+```
+
+##### Restoring a backup
```
Comment (0x6d69636b) [inline README.md:159]:
```suggestion
##### Restoring a Backup

```

**Review by 0x6d69636b — COMMENTED** [review]
Thank you, this is very useful for others! :) Just a minor detail, I use title case (AP style) for headers. Could you please fix that (I added suggestions)?

**Comment by d1nuc0m** [comment]
>  Just a minor detail, I use title case (AP style) for headers. Could you please fix that (I added suggestions)?

Done :)

**Pull request merged**